functions-and-scope.js

```diff
@@ -4,6 +4,20 @@
 
 const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 
+function gradesabove(array){
+    let count = null
+    for (let i = 0; i < array.length; i++) {
+        if (array[i] >=8){
+            count = count+1
```

Comment:
Je zou dit nog wat kunnen versimpelen
```suggestion
            count++;
```